boards/nxp/frdm_mcxn947/board.c

```diff
@@ -119,6 +127,11 @@ static int frdm_mcxn947_init(void)
 	CLOCK_EnableClock(kCLOCK_Gpio5);
 #endif
 
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(flexspi), okay)
+	/* Find the clock closest to the max frequency */
+	flexspi_clock_set_freq(0, DT_PROP(DT_NODELABEL(w25q64jvssiq), spi_max_frequency));
```

Comment:
Is there a reason we need to explicitly clock the FlexSPI  here? The flash driver should handle reclocking the FlexSPI if it is necessary, right?

Reply:
I have removed this and tested and it works.